src/main/java/io/swagger/oas/inflector/controllers/OpenAPIResourceController.java

```diff
@@ -71,7 +72,8 @@ public Response apply(ContainerRequestContext arg0) {
 
             MultivaluedMap<String, String> headers = arg0.getHeaders();
             // since https://github.com/swagger-api/swagger-inflector/issues/305 filtering of inflector extensions is handled at init time by ExtensionsUtils, and VendorSpecFilter is not needed anymore
-            return Response.ok().entity(getOpenAPI()).build();
+            return Response.ok().entity(new VendorSpecFilter().filter(getOpenAPI(), filter, null, cookies, headers)).build();
```

Comment:
I believe we better use (must use) `new SpecFilter` instead of `VendorSpecFilter` as `VendorSpecFilter` also always removes other extensions which presence since #305 is handled via config and extension utils

Reply:
Changed made.